ambry-frontend/src/test/java/com.github.ambry.frontend/FrontendIntegrationTest.java

```diff
@@ -207,6 +209,7 @@ private ByteBuffer getContent(Queue<HttpObject> contents, long expectedContentLe
     }
     assertEquals("Content length did not match expected", expectedContentLength, buffer.position());
     assertTrue("End marker was not found", endMarkerFound);
+    buffer.flip();
```

Comment:
Yep. All of the checks still use the array itself, but I felt this was good to add.